src/components/competence-overview/TeamContainer.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Team from "./Team";
+
+export default class TeamContainer extends React.Component {
```

Comment:
Very good! Making a core component that is stateless and wrapping it in a stateful component is a very useful pattern.